duckduckgo/iOS
# Reload page if it is current page and web content process is destroyed

Reviewer: Chris
Asana: https://app.asana.com/0/414235014887631/448081011607355

**Description**:
• updated the app to reload the current page if its web content process is destroyed
• improved memory logic by just invalidating an item's cache when the web content process is destroyed. There is no value in slimming the cache on an app memory warning as WKWebviews don't trigger the standard memory warning. The first sign of memory issues is the web content process being destroyed and we now remove those from the cache anyway.

**Steps to test this PR**:
1. Open a couple of tabs on an older device
2. Go to http://www.thezombiemudrun.com/
3. Zoom in and out and note that the screen freezes
4. The device will eventually run low on memory and the system will kill the content process. The page should recover and reload
5. Check that the other tabs are ok. If they too had their content process killed (very likely) they will have been removed from the cache and will be recreated when the user tries to open them

###### Reviewer Checklist:


###### PR DRI Checklist:
